bedrock/firefox/templates/firefox/browsers/mobile/includes/s2d-ios-sms.html

```diff
@@ -0,0 +1,51 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{# Send-to-device SMS widget for Firefox for iOS #}
+
+<div class="c-inline-form">
+  <h2 class="c-inline-form-title">{{ ftl('mobile-ios-get-firefox-for') }}</h2>
+  <p class="c-inline-form-body">{{ ftl('mobile-ios-send-a-download-link-to-your') }}</p>
+  <section id="s2d-sms" class="send-to-device vertical" data-spinner-color="#000" data-countries="US">
+    <div class="form-container">
+      <h2 class="mzp-u-title-xs thank-you hidden">{{ ftl('send-to-device-your-download-link') }}</h2>
+      <form class="send-to-device-form" action="{{ url('firefox.sms-send-to-device-post') }}" method="post">
+        <ul class="mzp-c-form-errors hidden">
+          <li class="number">Enter a 10-digit phone number</li>
+          <li class="sms">Sorry, we can’t send SMS messages to this phone number.</li>
```

Comment:
Should these strings be localized?

Reply:
The experiment is English-only, but if this becomes permanent we'll localize stuff.